.eslintignore

```diff
@@ -8,3 +8,4 @@ emscripten-sdk/*
 emsdk/*
 coverage/*
 objs/*
+gh-pages/*
```

Comment:
This is output that should be ignored when running `npm run doxygen` locally to generate docs